packages/devtools_app/lib/src/shared/config_specific/server/_server_stub.dart

```diff
@@ -102,6 +103,22 @@ Future<ExtensionEnabledState> extensionEnabledState({
 Future<List<String>> requestAndroidBuildVariants(String path) async =>
     const <String>[];
 
+Future<AppLinkSettings> requestAndroidAppLinkSettings(
+  String path, {
+  required String buildVariant,
+}) async =>
+    AppLinkSettings.empty;
+
+Future<XcodeBuildOptions> requestIosBuildOptions(String path) async =>
```

Comment:
should this be "variants" instead of "options" to match the android endpoint?

Reply:
In iOS, it is call build options. `variant` is a keyword in Android gradle build system.